ProofWidgets/Demos/Venn.lean

```diff
@@ -14,7 +14,7 @@ def Set (α : Type u) := α → Prop
 namespace Set
 
 /-- Membership in a set -/
-protected def Mem (a : α) (s : Set α) : Prop :=
+protected def Mem (s : Set α) (a : α) : Prop :=
```

Comment:
Hi @semorrison, what was the purpose of this change? It looks like the order [hasn't changed](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Set/Defs.html#Set.Mem) in mathlib.

Reply:
Ah I see, leanprover/lean4#5020.